Build.ps1

```diff
@@ -5,7 +5,8 @@
 )
 
 if ($isLocal){
-  $packageSuffix = "dev" + [datetime]::UtcNow.Ticks.ToString()
+  $dstring = Get-Date -Format "ddHHmmss"
+  $packageSuffix = "dev" + $dstring
```

Comment:
I'm assuming this is b/c it's hard to mentally parse the Ticks string? :-) 

Can you add `yyMM` to the front of this just to make it clearer exactly when it's from?

Reply:
yes, Was hard to decipher a long number every time.

Reply:
:) shouldn't need to be human readable, with the update flow, this shouldn't be an issue.

Reply:
This is flagged as resolved, but shouldn't be done within the scope of this changes. Please revert and we can discuss.

Reply:
This is now reverted